pkg/webhook/testdata/api/v3/repos/foo/bar/compare/1234...2345

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
This is a direct copy of `1234...5678` with all `test.sts.yaml`s replaced with `test.yaml`.